backend/utils/redis_client.py

```
@@ -0,0 +1,119 @@
+            redis_url = os.environ.get('REDIS_URL', 'redis://localhost:6379/0')
+            try:
+                self._client = redis.from_url(
+                    redis_url,
+                    decode_responses=True,
+                    socket_connect_timeout=5,
+                    socket_timeout=5
+                )
+                # 测试连接
+                self._client.ping()
+                print(f"✓ Redis 连接成功: {redis_url}")
+            except Exception as e:
```

Ruff (0.14.13)

Do not catch blind exception: `Exception`

(BLE001)